build-xcompiler-armhf.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+DEST_DIR=$1; DEST_DIR=${DEST_DIR:="/tmp"}
+./build-generic.sh \
+  "xcompiler" \
+  "${DEST_DIR?}" \
+  ./ct-ng.sh \
+    "/home/xcompiler/ct-ng" \
+    "conf/wandboard.config" \
+    "arm-unknown-linux-gnueabi" \
```

Comment:
actually the successful wandboard armhf one was created with this value in the few places where it's used, maybe it's generic enough for both?

as far as ct-ng is concerned, the name "arm-unknown-linux-gnueabi" doesn't matter because we hack the config for it so both armel and armhf use that same name